packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -534,6 +538,21 @@ class _NavigationIndicator extends StatelessWidget {
   /// If null, defaults to [ColorScheme.secondary].
   final Color? color;
 
+  /// The width of the container that holds in the indicator.
+  ///
+  /// Defaults to `64`.
+  final double width;
+
+  /// The height of the container that holds in the indicator.
+  ///
+  /// Defaults to `32`.
+  final double height;
+
+  /// The radius of the container that holds in the indicator.
+  ///
+  /// Defaults to `16`.
+  final double borderRadius;
```

Comment:
Nit: I think it might be more consistent to make this param of type `BorderRadius`

Reply:
what about having a shape parameter with the default being StadiumBorder?

Reply:
Done - used `BorderRadius`.